microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -35,25 +67,37 @@ To submit suggestions and feedback, enter an [Issue about the Add-ons API](https
 <!-- ====================================================================== -->
 ## Before you begin
 
-To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center, by creating API credentials. Use the following steps to create API credentials.
+To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center, by creating API credentials.  Use the following steps to create API credentials.
 
 1. Visit Microsoft Partner Center and sign in to the account that you used to publish an add-on.
 
 1. Under the **Microsoft Edge** program, select **Publish API**.
 
 1. On the **Publish API** page, click the **Create API credentials** button.  This step may take a few minutes to finish.
 
-   The API credentials have now been created; you've enabled or renewed the API.  The **Client ID**, **Client secret**, **Expiry date**, and **Access token URL** are now displayed on the Publish APIs page:
+   The API credentials have now been created; you've enabled or renewed the API.  The following items are now displayed on the **Publish API** page:
+   * **Client ID**
+   * **Client secret** (for REST Publish API v1) or **API key** (for REST Publish API v2)
+   * **Expiry date**
+   * **Access token URL** (for REST Publish API v1)
 
    ![The 'Publish API' page at Partner Center after clicking 'Create API credentials', now showing Client ID, Client Secret, and Auth Token URL](./using-addons-api-images/create-api-credentials-button.png)
```

Comment:
does this png need to be updated to show v2 ui?
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-3269#before-you-begin

Reply:
plan to keep v1 png , and add v1.1 png in dedicated tab
implemented tabsets to completely separate v1.1 info/steps vs v1

closing old comments.  added todo notes in .md; will create fresh Comment at each todo note